### PR TITLE
#2171 first update of documentation with new features and dependencies

### PR DESCRIPTION
**Pull request by rosiereflo**

### CODE REVIEW

**Comment by rosiereflo** [comment]
I didn't review the solver and model sections

**Comment by rosiereflo** [comment]
@gautierbureau I just saw a folder licenses, should I add files for libxml2 and powsybl dependencies?

**Comment by gautierbureau** [comment]
> @gautierbureau I just saw a folder licenses, should I add files for libxml2 and powsybl dependencies?

yes I think we are supposed to do it

File: documentation/configuringDynawo/DynawoInputFiles.tex [hunk 1]
```diff
@@ -36,23 +36,38 @@ \section{Overview}
...
 \item A file providing the dynamic description of the test grid to simulate: the dynamic data file (\textbf{.dyd} extension);
 \item A file providing the needed parameters of the simulation: the parameter file (\textbf{.par} extension);
+\end{itemize}
+The iidm file can be optional for small test cases where the whole system is represented in the dyd file (see paragraph \ref{sec:Run a simulation with no iidm file}).
+Other files are mandatory. \\
+
+Others optional files can be given as inputs:
+\begin{itemize}
 \item A file providing the list of the variables to plot at the end of the simulation: the curves file (\textbf{.crv} extension);
+\item A file providing the list of the variables for which we want to observe the final value: the final state value file (\textbf{.fsv} extension);
```
Comment (adriguir) [inline documentation/configuringDynawo/DynawoInputFiles.tex:52]: Maybe we can simplify this sentence: "A file providing the final values for a list of variables: the final state value file ..."

File: documentation/configuringDynawo/DynawoInputFiles.tex [hunk 2]
```diff
@@ -64,8 +79,11 @@ \section{Overview}
 \item A file providing the chronological list of events that happened during the simulation: component connection, disconnection, automata activation, etc (timeline.txt).
 \item A file providing the list of constraints that happened in the network during the simulation (constraints.txt);
 \item A file providing the static state of the network at the end of the simulation (output.iidm). This iidm file can be used to start another dynamic simulation;
+\item A binary file providing the dynamic state of the network at the end of the simulation (output.dmp). This file can be used to start another dynamic simulation;
 \item A file providing the value of the desired variables at each time step in order to visualize their evolution through time (curves.crv);
-\item A file providing information about how the simulation went (dynamo.log). This log file contains error messages if the simulation crashed;
+\item A file providing the value of the desired variables at at the final time step (finalStateValues.fsv);
```
Comment (adriguir) [inline documentation/configuringDynawo/DynawoInputFiles.tex:84]: at at : typo

File: documentation/installation/installation.tex [hunk 3]
```diff
@@ -18,7 +18,9 @@
 
 \chapter{Install procedure}
 
-\Dynawo is available on two platforms: \textbf{Linux} and \textbf{Windows}. You can either build it from sources on each platform or use binaries distributed on Github. If you are on \textbf{MacOS} you ca use a \hyperref[sec:docker]{Docker} solution.
+\Dynawo is available on two platforms: \textbf{Linux} and \textbf{Windows}.
+You can either build it from sources on each platform or use binaries distributed on Github.
+If you are on \textbf{MacOS} you can use a \hyperref[sec:docker]{Docker} solution.
```
Comment (adriguir) [inline documentation/installation/installation.tex:23]: Shouldn't operating system be better than platform?

File: documentation/installation/installation.tex [hunk 4]
```diff
@@ -88,14 +90,14 @@ \section{Building requirements}
 
 \subsection{Linux}
 
-\Dynawo has be tested on Linux platforms (Centos and Debian based) and provided that you can install system packages there should be no problem on other Linux distribution. It has also been tested on macOS (only on Mojave 10.14.5) and works, a complete procedure to help non-developers install it on this platform will be available soon.
+\Dynawo has be tested on Linux platforms (Centos, Debian and Ubuntu based) and provided that you can install system packages there should be no problem on other Linux distribution.
```
Comment (adriguir) [inline documentation/installation/installation.tex:93]: has been

**Review by adriguir — CHANGES_REQUESTED** [review]

File: documentation/configuringDynawo/DynawoInputFiles.tex (hunk 1 above)
Comment (rosiereflo) [reply documentation/configuringDynawo/DynawoInputFiles.tex]: Done

**Review by adriguir — APPROVED** [review]

**Pull request merged**